README.md

```diff
@@ -6,23 +6,28 @@
 ![Tensorflow](https://img.shields.io/badge/tensorflow-v2.9.0+-success.svg)
 [![Contributions Welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://github.com/keras-team/keras-cv/issues)
 
-KerasCV is a library of modular computer vision oriented Keras components.
-These components include models, layers, metrics, losses, callbacks, and utility
-functions.
+KerasCV is a library of modular, framework-agnostic computer vision components 
+that works natively with TensorFlow, JAX, or PyTorch. These components include 
+models, layers, metrics, losses, callbacks, and utility functions.
 
 <img style="width: 440px; max-width: 90%;" src="https://storage.googleapis.com/keras-cv/guides/keras-cv-augmentations.gif">
 
-KerasCV's primary goal is to provide a coherent, elegant, and pleasant API to train state of the art computer vision models.
-Users should be able to train state of the art models using only `Keras`, `KerasCV`, and TensorFlow core (i.e. `tf.data`) components.
+KerasCV can be understood as a horizontal extension of the Keras API: the 
+components are new first-party Keras objects (layers, metrics, etc.) that are 
+too specialized to be added to core Keras. They receive the same level of polish
+and backwards compatibility guarantees as the core Keras API, and they are 
+maintained by the Keras team.
 
-KerasCV can be understood as a horizontal extension of the Keras API: the components are new first-party
-Keras objects (layers, metrics, etc.) that are too specialized to be added to core Keras. They receive the same level of polish and backwards compatibility guarantees as the core Keras API, and they are maintained by the Keras team.
+Our APIs assist in common computer vision tasks such as data augmentation, 
+classification, object detection, segmentation, image generation, and more.
+Applied computer vision engineers can leverage KerasCV to quickly assemble 
+production-grade, state-of-the-art training and inference pipelines for all of 
+these common tasks.
 
-Our APIs assist in common computer vision tasks such as data-augmentation, classification, object detection, image generation, and more.
-Applied computer vision engineers can leverage KerasCV to quickly assemble production-grade, state-of-the-art training and inference pipelines for all of these common tasks.
-
-In addition to API consistency, KerasCV components aim to be mixed-precision compatible, QAT compatible, XLA compilable, and TPU compatible.
-We also aim to provide generic model optimization tools for deployment on devices such as onboard GPUs, mobile, and edge chips.
+With the release of Keras Core (link)[https://keras.io/keras_core/announcement/],
```

Comment:
Your link syntax is backwards. Make sure to view this rendered somewhere. One way is through the PR UI I think.

Also, I tend to prefer highlight the thing itself, e.g. With the [release of Keras Core](https://keras.io/keras_core/announcement/), but that's probably just a matter of preference.